packages/components/Tabs/src/TabsItem.tsx

```diff
@@ -0,0 +1,158 @@
+/** @jsx withSlots */
+import * as React from 'react';
+import { View } from 'react-native';
+import { compose, IUseComposeStyling } from '@uifabricshared/foundation-compose';
+import { ISlots, withSlots } from '@uifabricshared/foundation-composable';
+import { Text } from '@fluentui-react-native/text';
+import { Icon } from '@fluentui-react-native/icon';
+import { settings, tabsItemSelectActionLabel } from './TabsItem.settings';
+import { backgroundColorTokens, borderTokens, textTokens, foregroundColorTokens, getPaletteFromTheme } from '@fluentui-react-native/tokens';
+import { filterViewProps } from '@fluentui-react-native/adapters';
+import { mergeSettings } from '@uifabricshared/foundation-settings';
+import { TabsContext } from './Tabs';
+import { tabsItemName, TabsItemType, TabsItemProps, TabsItemSlotProps, TabsItemRenderData, TabsItemState } from './TabsItem.types';
+import {
+  useAsPressable,
+  useKeyCallback,
+  useViewCommandFocus,
+  createIconProps,
+  useOnPressWithFocus,
+} from '@fluentui-react-native/interactive-hooks';
+
+export const TabsItem = compose<TabsItemType>({
+  displayName: tabsItemName,
+
+  usePrepareProps: (userProps: TabsItemProps, useStyling: IUseComposeStyling<TabsItemType>) => {
+    const {
+      icon,
+      headerText = '',
+      onAccessibilityTap = userProps.onClick,
+      accessibilityLabel = userProps.headerText,
```

Comment:
Should this be `ariaLabel` instead?

Reply:
Actually, I see how `accessibilityLabel` works. Now I'm wondering what would happen if someone sets `ariaLabel` and `accessibilityLabel` on TabItem. Would the `ariaLabel` property work?

Reply:
Also, I think the count should be included in the accessibilityLabel if you're generating it from userProps

Reply:
^ I was about to ask the same questions. Note that if accessibilityLabel included count, I think we would then need some localization to get the right string format? i.e: ` "headerText + " " + count ` might not be the correct ordering for every language.

Reply:
When you say "might not be the correct ordering for every language" are you saying that in some languages people prefer to hear/read the number first then the item?

Reply:
> Actually, I see how `accessibilityLabel` works. Now I'm wondering what would happen if someone sets `ariaLabel` and `accessibilityLabel` on TabItem. Would the `ariaLabel` property work?

Currently it would not work. Should we keep accessibilityLabel or ariaLabel as the prop to pass to TabsItem?

Reply:
Yes. In general when an accessibilityLabel is a combination of multiple elements in code (Like a First Name + Last Name, label + count, x of y items, etc), it needs to be localized. In native apple code, we have a system for doing that. I think downstream SDX'es that use FURN have a system to localize their code as well. FURN doesn't quite yet, but it hasn't been a problem AFAIK because most of our labels have just been static text or roles like `tab` (which presumably would get localized by the OS). I'm not sure how it works on win32 but I'm sure Krystal is the right person to ask there

Reply:
> Should we keep accessibilityLabel or ariaLabel as the prop to pass to TabsItem?

I think we should use accessibilityLabel for both Tabs and TabsItem. I believe ariaLabel was added to RadioGroup/Button and Checkbox by @samuelfreiberg but they appear to be the same property. Sammy, can you remind me why ariaLabel was added as a property on those controls? Could accessibilityLabel have been used instead? Having both is a bit confusing.

Reply:
As for localization, I agree with everything Saad said. I don't have a good answer for how we'd localize it on the FURN side so I'll have to think more on that. But allowing the default value for accessibilityLabel to be overwritten will allow developers to have some control if the default value doesn't work for them.

Reply:
What is "count" here? Is that supposed to be `accessibilityPosInSet`?

I think if we set `accessibilityPosInSet`/`accessibilitySetSize` the screen readers will do the loc work for us

Reply:
@rurikoaraki "count" here refers to an item count you can add to the tab item text like:

![image](https://user-images.githubusercontent.com/22876140/128369316-c46d2d71-1f72-4b2f-bebb-44755b487188.png)

Reply:
@ksiler I don't quite remember why we decided on exposing `ariaLabel` instead of `accessibilityLabel` on RadioGroup/RadioButton/Checkbox, but I remember we had a conversation about it and that's what we agreed on during the spec review.

I'm ok with using accessibilityLabel here instead of ariaLabel, and then perhaps we can look into deprecating ariaLabel for accessibilityLabel in RadioGrou/Checkbox

Reply:
I set accessibilityLabel, accessibilityPosInSet, and accessibilitySetSize as the accessibility props.

Reply:
Yes, I just started a thread about it. I'll have a PR out shortly.